multimodal/src/autogluon/multimodal/utils/data.py

```diff
@@ -193,6 +202,7 @@ def create_fusion_data_processors(
         NUMERICAL: [],
         LABEL: [],
         NER: [],
+        ROIS: [],
```

Comment:
One thing to think about : are rois and ner are proper data types?

Reply:
ROIS should be data types, and we could enable it only in certain problem_types in the future if there is a conflict.